单步调试
python3 -m pdb my.py

或者
import pdb
pdb.set_trace()

https://blog.csdn.net/sgdd123/article/details/106126732
